app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,72 @@
+class Api::V1::UsersController < ApplicationController
+  before_action :authorized, only: [:load_user, :show]
+  before_action :load_user, only: [:show]
+
+  def show
+    render json: {
+      code: 200,
+      message: :ok,
+      data: @user
+    }, status: :ok
+  end
+
+  # REGISTER
+  def create
+    @user = User.create(user_signup_params)
```

Comment:
bỏ ngoặc các hàm có thể bỏ
```suggestion
    @user = User.create user_signup_params
```